.github/workflows/epub-a11y-tech.yml

```diff
@@ -23,5 +23,5 @@ jobs:
           W3C_ECHIDNA_TOKEN: ${{ secrets.W3C_TR_TOKEN_A11Y_TECH }}
           W3C_WG_DECISION_URL: https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-02-12-epub#resolution2
```

Comment:
```suggestion
          W3C_WG_DECISION_URL: https://lists.w3.org/Archives/Public/public-pm-wg/2023Oct/0028.html
```

Reply:
Ah, right, perfect! I was thinking it needed to be updated with each new FPWD but this is the decision to publish by echidna not to publish new versions.